src/DynamoCore/Nodes/Custom Nodes/Function.cs

```diff
@@ -315,6 +317,33 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
             }
 
             ArgumentLacing = LacingStrategy.Disabled;
+
+            DeserializeElementResolver(nodeElement);
+        }
+
+        private void DeserializeElementResolver(XmlElement nodeElement)
```

Comment:
This seems to duplicate the above code, please refactor them into one utility method and add test cases.